packages/synthetics-sdk-broken-links/test/unit/link_utils.spec.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {expect} from 'chai';
+import {
+  ResponseStatusCode,
+  ResponseStatusCode_StatusClass,
+} from '@google-cloud/synthetics-sdk-api';
+const BrokenLinksUtils = require('synthetics-sdk-broken-links-utils');
```

Comment:
If this is the only use case for using synthetics-sdk-broken-links-utils, use the relative path instead.

Reply:
ok!